### PR TITLE
User data directory

### PR DESCRIPTION
**Pull request by jcollado**
In this PR, the [`appdirs`](https://pypi.python.org/pypi/appdirs) package is used to get a customized user data directory that depends on the OS instead of getting a hardcoded default value.

### CODE REVIEW

File: ghost.py [hunk 1]
```diff
@@ -30,6 +30,7 @@
 from datetime import datetime
 
 import click
+from appdirs import user_data_dir
```
Comment (nir0s) [inline ghost.py]: Ordering imports by length
please move this one down.

Thanks

File: setup.py [hunk 2]
```diff
@@ -23,6 +23,7 @@ def read(*parts):
     py_modules=['ghost'],
     entry_points={'console_scripts': ['ghost = ghost:main']},
     install_requires=[
+        "appdirs==1.4.0",
```
Comment (nir0s) [inline setup.py]: Again, for maintaining order, please move this down

**Comment by nir0s** [comment]
Please also update the documentation according to the changes (for Linux only)

File: ghost.py (hunk 1 above)
Comment (jcollado) [inline ghost.py]: Fixed in ef2223f

File: setup.py (hunk 2 above)
Comment (jcollado) [inline setup.py]: Fixed in ef2223f

**Comment by jcollado** [comment]
I believe all the suggested changes have been applied. Please let me know if there's something else that is missing.

**Comment by nir0s** [comment]
Please rebase so that I can merge

**Pull request merged**